android/app/src/main/java/org/fossasia/openevent/utils/Views.java

```diff
@@ -194,4 +197,10 @@ public static void setEdgeGlowColor(@NonNull EdgeEffectCompat edgeEffect, @Color
         if (effect != null)
             effect.setColor(color);
     }
+
+    public static int getAccentColor(final Context context) {
+        final TypedValue value = new TypedValue();
+        context.getTheme().resolveAttribute(R.attr.colorAccent, value, true);
+        return value.data;
+    }
```

Comment:
You can use `ContextCompat.getColor(context, R.color.colorAccent)` instead

Reply:
Ok